source/iterators.tex

```diff
@@ -6078,12 +6078,17 @@
 iterator value.
 The default constructor
 \tcode{istreambuf_iterator()}
-and the constructor
-\tcode{istreambuf_iterator(0)}
-both construct an end-of-stream iterator object suitable for use
+and the constructors
+\tcode{istreambuf_iterator(default_sentinel)}
+and
+\tcode{istreambuf_iterator(nullptr)}
+all construct an end-of-stream iterator object suitable for use
 as an end-of-range.
-All specializations of \tcode{istreambuf_iterator} shall have a trivial copy
-constructor, a \tcode{constexpr} default constructor, and a trivial destructor.
+All specializations of \tcode{istreambuf_iterator} shall have
+a trivial copy constructor,
+a \tcode{constexpr} default constructor,
+a \tcode{constexpr} constructor from \tcode{default_sentinel},
```

Comment:
Is this worded right?

Reply:
Dropping fc413fb in favour of a better editorial solution similar to that of https://wg21.link/P0738.